### DISCUSSION TITLE
Copying multiple cells [Jan 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4902] [Discord Threads]
> 🙋‍♂️ **ITYPE**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### 1. if you copy multiple cells and paste to other cells, only the first one will be pasted. however, if u copy them and paste to a editor (all copied data should be there)   
#### 2. need more info on this part. there are different cases. it should show the message when you paste it.  
#  
> 🙋‍♂️ **ITYPE**  
#### I need to copy 10 cells of one column into 10 cells of different column. It's not working.  
#  
> 🙋‍♂️ **wkw**  
#### if you copy multiple cells and paste to other cells, only the first one will be pasted.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1052874437537579010)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
